### ISSUE TITLE
Support Coefficient Packing in OpenFHE

### ISSUE THREAD

**Issue by AlexanderViand-Intel** [google/heir#1192]
Probably requires a more explicit packing layout first (e.g., #913).

OpenFHE offers `MakeCoefPackedPlaintext` and we've got `lwe.polynomial_coefficient_encoding`, but right now we have neither a `openfhe.make_coef_packed_plaintext` operation to codegen/emit, nor would the compiler know how to deal with such ciphertexts.
For example, `rotate` takes on a very different meaning for such a ciphertext (multiply with $X$ as in CGGI, not the complex automorphism-based rotate used with most other packed ciphertexts).

**Comment by j2kun** [comment]
Is rotate even desirable for such encodings? While I'm happy to have support for coefficient packing, it may be safer to raise an error if someone tries to rotate a coefficient packed ciphertext.

**Comment by AlexanderViand-Intel** [comment]
Yeah, I think raising an error is a good starting point, though I assume we eventually want to support the "multiply with $X$" rotation on such packed ciphertexts for CGGI-stuff/scheme switching anyway.
